src/Generator.cpp

```diff
@@ -128,7 +128,7 @@ int generate_filter_main(int argc, char **argv, std::ostream &cerr) {
 
 GeneratorParamBase::GeneratorParamBase(const std::string &name) : name(name) {
     ObjectInstanceRegistry::register_instance(this, 0, ObjectInstanceRegistry::GeneratorParam,
-                                              this);
+                                              this, nullptr);
```

Comment:
The plan is still to wait for the release of the next Visual Studio
version, so that two versions of it support C++11. Looks like the preview
is out, so it should be soon.

On Thu, Jan 15, 2015 at 11:14 AM, Steven Johnson <notifications@github.com>
wrote:

> In src/Generator.cpp
> <https://github.com/halide/Halide/pull/628#discussion-diff-23034950>:
>
> > @@ -128,7 +128,7 @@ int generate_filter_main(int argc, char **argv, std::ostream &cerr) {
> >
> >  GeneratorParamBase::GeneratorParamBase(const std::string &name) : name(name) {
> >      ObjectInstanceRegistry::register_instance(this, 0, ObjectInstanceRegistry::GeneratorParam,
> > -                                              this);
> > +                                              this, nullptr);
>
> It's 2015. Can haz C++[$CURRENT_YEAR-4]?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/halide/Halide/pull/628/files#r23034950>.
>